src/ArborX_BruteForce.hpp

```diff
@@ -0,0 +1,124 @@
+/****************************************************************************
+ * Copyright (c) 2012-2019 by the ArborX authors                            *
```

Comment:
Fix years in all newly added files
```suggestion
 * Copyright (c) 2017-2021 by the ArborX authors                            *
```

Reply:
Fixed